pkg/strategy/dca2/strategy.go

```diff
@@ -146,6 +168,15 @@ func (s *Strategy) Run(ctx context.Context, _ bbgo.OrderExecutor, session *bbgo.
 		s.Position = types.NewPositionFromMarket(s.Market)
 	}
 
+	// prometheus
```

Comment:
add a meaningful comment? like... avoid registering metrics twice and override the previous one?